### ISSUE TITLE
LiveApp not support audio/G711.alaw?

### ISSUE THREAD

**Issue by 14780001553** [ant-media/Ant-Media-Server#3532]

**Comment by mekya** [comment]
Yes G711 is not supported. It supports Opus for WebRTC streaming. 

Why do you need G.711 support?

**Comment by babutree** [comment]
Sorry, In China most of IP Camera on desk doesn't support ONVIF protocol, I use Mercury MIPC 451-4. It's support ONVIF, but its audio encode only support G.711 protocol.

2023-03-03 06:40:04,972 [Thread-169] INFO  i.a.streamsource.StreamFetcher - Setting rtsp transport type to tcp for stream source: rtsp://admin:123456789@192.168.1.198:554/stream1
2023-03-03 06:40:04,972 [Thread-169] ERROR i.a.streamsource.StreamFetcher - cannot open stream: rtsp://admin:123456789@192.168.1.198:554/stream1 with error:: Invalid argument
2023-03-03 06:40:04,972 [Thread-169] ERROR i.a.streamsource.StreamFetcher - Prepare for opening the rtsp://admin:123456789@192.168.1.198:554/stream1 has failed
2023-03-03 06:40:04,977 [Thread-169] INFO  i.a.streamsource.StreamFetcher - write all buffered packets for stream: 1jKS1tJ2fc1t1677824726004
2023-03-03 06:40:04,977 [Thread-169] INFO  i.a.streamsource.StreamFetcher - Stream fetcher will try to fetch source rtsp://admin:123456789@192.168.1.198:554/stream1 after 3000 ms
2023-03-03 06:40:07,982 [Thread-170] INFO  i.a.streamsource.StreamFetcher - Preparing the StreamFetcher for rtsp://admin:123456789@192.168.1.198:554/stream1 for streamId:1jKS1tJ2fc1t1677824726004

![image](https://user-images.githubusercontent.com/21240045/222649631-84a33cd3-ce0c-4f9f-b6e7-35c831c3108c.png)

**Comment by mekya** [comment]
Hi @babutree ,

The problem may not be the G711A-law support in WebRTC streaming. 
It seems that the server cannot open the RTSP URL in your IP camera. 

I think the problem should be something else. Could you please write the ffmpeg output of your command?
```
ffmpeg -i rtsp_transport tcp -i {YOUR_RTSP_URL}
```

**Comment by ReapmindInvo** [comment]
@mekya I have found the same issue and reported it in https://github.com/ant-media/Ant-Media-Server/issues/6856

Camera configuration also is the same.

**Comment by SahilKalaigar** [comment]
I am facing same issue for Ant Media Server Community Edition 2.11.3 20240906_1141
